### ISSUE TITLE
help output mixes MB and GB for memory argument

### ISSUE THREAD

**Issue by tokeefe** [nipreps/fmriprep#2440]
`fmriprep --help` mixes `MB` and `GB` for memory argument

```
--mem MEMORY_GB, --mem_mb MEMORY_GB, --mem-mb MEMORY_GB
                        upper bound memory limit for fMRIPrep processes
                        (default: None)
```

**Comment by oesteban** [comment]
Thanks for reporting this. `--mem_mb` and `--mem-mb` are legacy flag names and we should definitely discontinue them (or at least, change them for corresponding `gb` versions.

**Comment by effigies** [comment]
Should have been fixed in #2738.

**Comment by effigies** [comment]
@oesteban I would suggest closing this and leaving it as `--mem-mb`.
